client/k8srestconfig/k8s_rest_config.go

```diff
@@ -93,10 +95,11 @@ type Config struct {
 	Logger micrologger.Logger
 
 	// Settings
-	Address   string
-	InCluster bool
-	Timeout   time.Duration
-	TLS       TLSClientConfig
+	Address    string
+	KubeConfig string
+	InCluster  bool
+	Timeout    time.Duration
+	TLS        TLSClientConfig
```

Comment:
I noticed that `TLSClientConfig` should be `ConfigTLS`.

Reply:
Nice catch! Fixed.

Reply:
Just be aware that this is a breaking change and we should have at least a notice in some news channel so we can fix it when we update vendor in a project.